src/app/_components/CategorySidebar.tsx

```diff
@@ -17,10 +13,9 @@ const touchTarget = {
 } as const
 
 type CategorySidebarProps = {
-  selectedId: CategoryId
-  options: Array<CategoryOption>
-  counts?: CategoryCounts
-  onChange: (selected: CategoryOption) => void
+  selectedId: OptionType['id']
+  options: Array<OptionType>
+  onChange: (selected: OptionType) => void
```

Comment:
We remove the `counts` props are we calculate and add it to `options` object in the parent component